src/main/java/io/confluent/idesidecar/restapi/resources/ConnectionsResource.java

```diff
@@ -38,6 +46,7 @@
 @Consumes(MediaType.APPLICATION_JSON)
 @Blocking
 public class ConnectionsResource {
+  private static final Logger logger = LoggerFactory.getLogger(ConnectionsResource.class);
```

Comment:
Let's import `import io.quarkus.logging.Log` and then directly use it with `Log.<whatever>`. Refer to other code files for similar usage.